labot/sniffer/ui.py

```diff
@@ -113,6 +115,7 @@ def async_start():
     loop = asyncio.get_event_loop()
     t = threading.Thread(target=loop_in_thread, args=(loop,))
     t.start()
+    t.join()
```

Comment:
I'm not sure about what you are doing here. Aren't you making this function blocking?

Reply:
Ah, sorry, that slipped through in the commit... I don't know why, but when I run the sniffer as a module (from runsniffer.py), execution stops immediately... I know almost nothing of threading in Python so this is the only way I found to make it work :/

Reply:
Just do

    python -m labot.sniffer.main

not the ideal way, but I didn't have better ideas. What is runsniffer.py?

Reply:
Yes, but unfortunately I can't debug it with Visual Studio Code that way.

runsniffer.py adds that possibility (it wasn't even meant to be part of the pull request, just for me to be able to debug, but then I thought other people might have the same issue).

Reply:
That explains the absolute imports.

Maybe you can create a new file runsniffer.py with

    from labot.sniffer.main import *